mkdocs.yml

```diff
@@ -62,7 +63,8 @@ theme:
   favicon: assets/ms_icon.png
   features:
     - navigation.instant
-    - navigation.tabs
+    # - navigation.tabs
```

Comment:
Intentional?

Reply:
Yes, the tabs above are hidden if you scroll. The only way to get them to stay is to upgrade mkdocs-material to the Insiders version.